platform/mellanox/get_component_versions/get_component_versions.j2

```diff
@@ -136,13 +154,17 @@ def main():
         actual = get_current_version(comp)
         output_table.append([comp, compiled_versions[comp], actual])
 
+    {% if sonic_asic_platform == "mellanox" %}
```

Comment:
This if statement is not needed. For the DPU `DeviceDataManager.is_simx_platform()` will return False

Reply:
I got an error saying DeviceDataManager has no method for is_simx_platform() when I was building for the dpu. Hence added this

Reply:
Updated